service/build.gradle

```diff
@@ -149,13 +152,20 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop', version: versions.springBoot
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-json', version: versions.springBoot
 
+  compile 'org.json:json:20171018'
```

Comment:
Is that really needed? If so define in `compile group: '', name: '', version: ''` format.